html-forms/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML FORMS</title>
+  </head>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <label for="email"> Email </label>
+    <div>
+      <input type="email" name="email" id="email" required />
+    </div>
+
+    <div>Why is coding cool?</div>
```

Comment:
This text should be in a `label`

Reply:
Committed in wrong branch previously however most recent commit is the correct branch.